test/acceptance/features/bindAppToService.feature

```diff
@@ -328,16 +328,13 @@ Feature: Bind an application to a service
                     kind: Backend
                     name: backend-demo-empty-app
             """
-        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-empty-app" should be changed to "True"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-empty-app" should be changed to "False"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-empty-app" should be changed to "EmptyApplication"
-        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-empty-app" should be changed to "True"
-        And Secret contains "BACKEND_HOST" key with value "example.common"
-        And Secret contains "BACKEND_USERNAME" key with value "foo"
+        Then Error message is thrown
+        And Service Binding "binding-request-empty-app" is not persistent in the cluster
 
     @olm
     Scenario: Backend Service new spec status update gets propagated to the binding secret
         Given OLM Operator "backend-new-spec" is running
+        * Generic test application "gen-app-a-s-c" is running
```

Comment:
this scenario should be removed because we do not support just creating binding secrets if application is not specified.

Reply:
This scenario is different, please read the scenario once again. The empty application was used earlier for convenience.

Reply:
The scenario description is "Backend Service new spec status update gets propagated to the binding secret" and IMHO it is obsolete now because now we have introduced the application ref in the whole game. Hence, the scenario description should be renamed to reflect that, e,g,:

"Bind service to application using binding definition available in x-descriptors"

And to assert that, we should drop lines 372-374 and check instead what application sees.

Reply:
Done. 29d348dd24be04f7ef01892df9186f888247adf1